src/System.Private.CoreLib/shared/System/IO/Path.Unix.cs

```diff
@@ -197,17 +199,25 @@ public static string GetPathRoot(string path)
             return IsPathRooted(path) ? PathInternal.DirectorySeparatorCharAsString : String.Empty;
         }
 
+        private static unsafe void GetCryptoRandomBytes(byte* bytes, int byteCount)
+        {
+            // We want to avoid dependencies on the Crypto library when compiling in CoreCLR. This
+            // will use the existing PAL implementation.
+            byte[] buffer = new byte[KeyLength];
+            Microsoft.Win32.Win32Native.Random(bStrong: true, buffer: buffer, length: KeyLength);
```

Comment:
This will not work on CoreRT Unix. I think we want to keep calling Interop.GetRandomBytes in CoreRT, and add an implementation of it to CoreCLR that will call CoreCLR PAL - at least for now.